packages/cozy-harvest-lib/README.md

```diff
@@ -69,6 +69,26 @@ Cozy-Harvest-Lib provides a defined range of field labels which do not need to b
 
 Any konnector can provide new locales for this field labels or add new ones. They will be loaded by the AccountForm component.
 
+#### `default placeholder`
```

Comment:
What do we want here is explain how the placeholder are working :

* No placeholder by default
* Placeholders may be specified in the locales object of the konnectors manifest : give an example